config/src/test/java/org/springframework/security/config/web/server/OneTimeTokenLoginSpecTests.java

```diff
@@ -280,6 +286,36 @@ Please provide it as a bean or pass it to the oneTimeTokenLogin() DSL.
 					""");
 	}
 
+	@Test
+	void oneTimeTokenWhenCustomTokenExpirationTimeSetThenAuthenticate() throws Exception {
+		this.spring.register(OneTimeTokenConfigWithCustomTokenExpirationTime.class).autowire();
+
+		// @formatter:off
+		this.client.mutateWith(SecurityMockServerConfigurers.csrf())
+				.post()
+				.uri((uriBuilder) -> uriBuilder
+						.path("/ott/generate")
+						.build()
+				)
+				.contentType(MediaType.APPLICATION_FORM_URLENCODED)
+				.body(BodyInserters.fromFormData("username", "user"))
+				.exchange()
+				.expectStatus()
+				.is3xxRedirection()
+				.expectHeader().valueEquals("Location", "/login/ott");
+		// @formatter:on
+
+		OneTimeToken token = TestServerOneTimeTokenGenerationSuccessHandler.lastToken;
```

Comment:
This will likely cause issues with concurrent tests. 

Please see https://github.com/spring-projects/spring-security/commit/751b5580a129f9b3a1da02c7f49bffd6ed35ef32 - will you please make a similar repair in a preceding commit?